pettingzoo/classic/chess/chess_utils.py

```diff
@@ -194,7 +194,7 @@ def legal_moves(orig_board):
     return legal_moves
 
 
-def get_observation(orig_board, player):
+def get_observation(orig_board, player: int):
```

Comment:
If you don't mind it would actually be nice to have type hints for all of this file

Reply:
I add a few type hints if it is obvious, e.g. `chess.Board`.